vaccine/tests/test_vaccine_reg_ussd.py

```diff
@@ -988,19 +988,21 @@ async def test_medical_aid_invalid():
 @pytest.mark.asyncio
 async def test_terms_and_conditions():
     u = User(
-        addr="27820001001", state=StateData(name="state_medical_aid"), session_id=1
+        addr="27820001001",
+        state=StateData(name="state_terms_and_conditions_3"),
```

Comment:
This should be the state before `state_terms_and_conditions`, so that when you answer that state, it displays `state_terms_and_conditions`. In this case, it's `state_age_gate`